packages/example-getting-started/docs/3-add-legacy-juggler.md

```diff
@@ -1,5 +1,15 @@
 ### Adding the Legacy Juggler
 
+The Legacy Juggler is a "bridge" between the existing
+[loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler)
+and the new LoopBack 4 architecture. It provides the capabilities required to
+access datasources, and perform
+[CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations
+on those datasources.
+
+It also provides many of the functions and interfaces we'll require for setting
+up our new LoopBack application, which is why we're starting here.
+
 Jump into the directory for your new application. You'll see a folder structure
 similar to this:
 ```
```

Comment:
@shimks Could you be more specific as to what you're expecting to see?

Reply:
I changed my mind. I originally wanted descriptions for the artifacts in the previous page and thought the folder structure diagram would go nicely with it, but since we're not doing that this is fine as it is I think.